internal/activitypub/person.go

```diff
@@ -2,20 +2,10 @@ package activitypub
 
 import (
 	"fmt"
-	"sublinks/sublinks-federation/internal/lemmy"
+	"sublinks/sublinks-federation/internal/model"
 	"time"
 )
 
-type PublicKey struct {
-	Keyid        string `json:"id"`
-	Owner        string `json:"owner"`
-	PublicKeyPem string `json:"publicKeyPem"`
-}
-
-type Endpoints struct {
-	SharedInbox string `json:"sharedInbox"`
-}
-
 type User struct {
```

Comment:
Oops... missed renaming this to Person. Will fix either in next PR or if there are any other changes requested

Reply:
Fixed